etc/updateOracle.sql

```diff
@@ -139,3 +139,22 @@ ALTER TABLE tasks ADD tm_publish_rule VARCHAR(255);
 
 --Add Rucio ASO's json kv for store container names and its rules.
 ALTER TABLE tasks ADD tm_multipub_rule CLOB;
+
+-- 2024-03 oracle cleanup
```

Comment:
I am not sure that it is worth to keep here (ever if as comments, IIUC) actions that you did just once to remove temporary but left-over tables. Maybe simply a note about this process in the doc ? (I have not read doc yet, maybe it is there already)

Reply:
oook! :)